index.js

```diff
@@ -42,7 +42,10 @@ function init() {
     const restoredNotificationsCheckboxValue = localStorage.getItem('notifications');
     formCheckboxElem.checked = JSON.parse(restoredNotificationsCheckboxValue);
     const restoredLang = localStorage.getItem('select-language');
-    changeLanguage(restoredLang);
+    const defaultLanguage = 'en';
```

Comment:
please, use UPPER_CASE_SNAKE for these kind of constants

Reply:
No. You didn't

Reply:
Sorry. Already done. Thanks.